media/admin/css/base.css

```diff
@@ -0,0 +1,998 @@
+/*
+    DJANGO Admin styles
```

Comment:
@eric38383 I'd recommend an approach to loading the new styles on the template, which should avoid duplicating the default styles.

Reply:
Like put it in the style tag?

Reply:
@eric38383 Yeah. Pointed to `{% static 'path/to/style.css %}`